src/sentry/features/__init__.py

```diff
@@ -126,6 +126,7 @@
 default_manager.add("organizations:device-classification", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
 default_manager.add("organizations:device-class-synthesis", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
 default_manager.add("organizations:monitors", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
+default_manager.add("organizations:monitors-quota-rate-limit", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
```

Comment:
don't you need to add this to `server.py`?

Reply:
Yes good catch. It's not required but it's good practice.

We really just need a test to enforce this